code/game/area/Space_Station_13_areas.dm

```diff
@@ -858,6 +854,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Teleporter Room"
 	icon_state = "teleporter"
 	music = "signal"

```

Comment:
Does each subtype of /area/teleporter need `music = "signal"`, or can that be cleaned up?

Reply:
Doubt it, a lot of code seemed to have copy pasta variables.  I'll go remove some of that